screens/SignupScreen.tsx

```diff
@@ -10,40 +13,89 @@ import Colors from '../theme/Colors';
 
 import { StyledContainer, StyledWideContainer } from './SignupScreen.styled';
 
+const SignupSchema = Yup.object().shape({
+  email: Yup.string()
+    .email('Invalid email')
+    .required('Required'),
+  phone: Yup.string()
+    .min(9, 'Too Short!')
+    .min(12, 'Too Short!'),
```

Comment:
Is this rule correct? (2 times `min()` ?)

Reply:
Thx for comment, I will fix it